.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+* @Arriven
+# /docs/ @iamtodor @Amet13
```

Comment:
Commented due to lack of permission for review.
But we can keep comments for information who to ask

Reply:
@Amet13 perhaps @Arriven could provide specific access according to the `least privilege principle`?

Reply:
I'll take a look, can't remember if regular user repos allow access differentiation